lib/addax-rdbms/src/main/java/com/wgzhao/addax/rdbms/writer/util/WriterUtil.java

```diff
@@ -173,6 +173,21 @@ else if (dataBaseType == DataBaseType.SQLServer) {
     private static String doPostgresqlUpdate(String writeMode, List<String> columnHolders)
     {
         String conflict = writeMode.replaceFirst("update", "").trim();
+        if(StringUtils.isEmpty(conflict)){
```

Comment:
The string of conflict is combined in code instead external passing. so  ,it not necessary to validate